pulsar-client-cpp/tests/BasicEndToEndTest.cc

```diff
@@ -1708,7 +1708,7 @@ TEST(BasicEndToEndTest, testSeekOnPartitionedTopic) {
     ASSERT_EQ(expected.str(), msgReceived.getDataAsString());
     ASSERT_EQ(ResultOk, consumer.acknowledge(msgReceived));
     ASSERT_EQ(ResultOk, consumer.unsubscribe());
-    ASSERT_EQ(ResultOk, consumer.close());
+    ASSERT_EQ(ResultAlreadyClosed, consumer.close());
```

Comment:
In the current logic, when unsubscribe is invoked, the status is modified to close.